machine/v1beta1/types_awsprovider.go

```diff
@@ -320,3 +328,18 @@ type AWSMachineProviderStatus struct {
 	// +optional
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 }
+
+// MarketType describes the market type of an Instance
+// +kubebuilder:validation:Enum:=OnDemand;Spot;CapacityBlock
+type MarketType string
+
+const (
+	// MarketTypeOnDemand is a MarketType enum value
```

Comment:
This applies to all 3 constants, but it would be good to put the same level of detail here as you did for each of the enum values on their corresponding constants. It is helpful from the developer side that may be using these constants in code and being able to use the LSP support to see the GoDoc on these.

Reply:
ack